start_state: [0.0, 0.0, 0.85,
              0.0, 0.0, 0.0,
              0.0, 0.0, 0.0]

waypoints: [ -3.0, 17.0, 1.0,
             5.0, 30.0, 2.0,
             -5.0, 42.0, 1.0,
             1.5, 55.0, 1.0]
# height=constant             
# waypoints: [ -2.0, 15.0, 0.85,
#              3.8, 30.0, 0.85,
#              -4.0, 40.0, 0.85,
#              0.0, 50.0, 0.85,
#              2.5, 60.0, 0.85]
# waypoints: [ -0.5, 10.0, 2.5,
#              3.6, 19.0, 0.5,
#              -3.2, 27.0, 2.5,
#              3.0, 40.0, 2.5,
#              2.5, 50.0, 0.5,
#              -3.0, 60.0, 2.5
#              ]

# zhangjiajie
# waypoints: [ -0.5, 10.0, 2.5,
#              3.6, 19.0, 0.5,
#              -3.2, 27.0, 2.5,
#              3.0, 40.0, 2.5,
#              2.5, 50.0, 0.5,
#              -3.0, 60.0, 2.5
#              ]

# circle
# waypoints: [ 8.0, 3.0, 1.0,
#              3.0, 8.0, 1.0,
#              -2.0, 3.0, 1.0,
#              3.0, -2.0, 1.0
#              ]

# waypoints: [ 2.5, 15.0, 2.5,
#              -3.0, 30.0, 0.5,
#              2.5, 40.0, 2.5,
#              2.5, 50.0, 0.5,
#              -3.0, 60.0, 2.5]

# goal_state: [2.5, 65.0, 2.5, 
#              0.0, 0.0, 0.0, 
#              0.0, 0.0, 0.0]

goal_state: [0.0, 66.0, 0.85, 
             0.0, 0.0, 0.0, 
             0.0, 0.0, 0.0]


# time_vector: [4.0, 4.0, 3.0, 3.0, 3.0, 4.0]
# time_vector: [4.0, 4.0, 4.0, 4.0, 4.0]
time_vector: [3.0, 2.0, 2.0, 2.0, 2.0]
# time_vector: [3.0, 3.0, 3.0, 3.0, 3.0, 2.0]
# time_vector: [12.0, 12.0, 9.0, 9.0, 9.0, 12.0]